pyproject.toml

```diff
@@ -124,6 +124,17 @@ tracker = "https://github.com/scipy/scipy/issues"
 [tool.doit]
 dodoFile = "dev.py"
 
+[tool.meson-python]
+meson = 'vendored-meson/meson/meson.py'
+
+[tool.meson-python.args]
+install = ['--tags=runtime,python-runtime,tests,devel']
+
+[tool.spin]
+package = 'numpy'
```

Comment:
This should presumably be `scipy` instead of `numpy`? Both seem to work locally with `spin test -j 32`, so not sure what this does.